reggie/ingestion/preprocessor/washington_preprocessor.py

```diff
@@ -91,12 +112,11 @@ def execute(self):
         )
 
         # --- handling the voter file --- #
-
```

Comment:
I know we talked about it but just leaving this here for reference - the sorted_elections_dict date format above needs to be "%m/%d/"%Y"